scripts/check_deployed_image_findings

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+set -u
+
+repository="$1"
+function_qualifier="$2"
+
+case "$APP_ENV" in
+  "dev")
+    EASI_URL="https://dev.easi.cms.gov"
+    ;;
+  "impl")
+    EASI_URL="https://impl.easi.cms.gov"
+    ;;
+  "prod")
+    EASI_URL="https://easi.cms.gov"
+    ;;
+  *)
+    echo "APP_ENV value not recognized: ${APP_ENV:-unset}"
+    echo "Allowed values: 'dev', 'impl', 'prod'"
+    exit 1
+    ;;
+esac
+
+# Get image version currently deployed
+image_tag=$(curl --silent "$EASI_URL"/api/v1/healthcheck | jq -r .version)
+
+# Check scan findings for image
+payload="$(jq --null-input ".repository= \"$repository\" | .imageTag = \"$image_tag\"")"
+aws lambda invoke --function-name ecr-scan-findings --cli-binary-format raw-in-base64-out --payload "$payload" --qualifier "$function_qualifier" scan_response.json
+jq < scan_response.json
+totalFindings="$(jq "fromjson | .totalFindings" scan_response.json)"
+
+# Quit if there were no findings
+if [[ "$totalFindings" = 0 ]]; then
+  exit 0
+fi
+
+# If there are findings, send a message to the Slack channel
+NOW=$(date '+%s')
+
+pretext="Image scan findings"
+title="Image scan findings detected in $APP_ENV"
+message="$totalFindings finding(s) detected in image $image_tag. Contact infra for more information."
+
+#####
+## Announce in Slack channel
+#####
+# 'color' can be any hex code or the key words 'good', 'warning', or 'danger'
+color="warning"
+
+slack_payload=$(
+cat <<EOM
```

Comment:
I'm nervous about this because this isn't a safe way to assemble JSON. What will happen if the JSON generated here is invalid? Would we get some other kind of error or no?

Reply:
Good question! I borrowed this code snippet from our `circleci-announce-broken-branch` script, so whatever updates we make here we should make there as well. Would you recommend the `jq --null-input` approach here?

> Would we get some other kind of error or no?

I think the short answer is no. We don't currently have the "announce failure" step configured for the impl/prod restart jobs, but we could add it:

```
      - run:
          name: Announce failure
          command: |
            ./scripts/circleci-announce-broken-branch
          when: on_fail
```

Reply:
I think that would be fine, but there may be alternatives. I like `jq` because you either get valid JSON or an error that you have a chance to handle.

> I think the short answer is no

I'm mostly worried that there would be findings, but because this script can emit invalid json, whatever reads the output from this script would choke and report nothing (and we'd be none the wiser).

Reply:
> I think that would be fine, but there may be alternatives.

Where there any alternatives you had in mind?

Reply:
You'd mentioned the `--arg` pattern in the back channel. Maybe that would work, but I'm not familiar with it. The upshot of what I'm saying is that assembling JSON in any way with `jq` is better than using simple string concatenation, in my opinion.

Reply:
Constructing the json with the `jq` `--arg` pattern is working https://github.com/CMSgov/easi-app/pull/747/commits/17b85c551c61907c062bbf6c6034819e7b03bb08

I also added the announce failure to step to the impl/prod restart jobs.